### PR TITLE
[Sample Update] Tab device permission sample readme changes

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**
Readme changes to reflect latest code updates

### CODE REVIEW

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments

File: samples/tab-device-permissions/nodejs/README.md [hunk 1]
```diff
@@ -20,10 +20,7 @@ This sample shows [tab device permissions](https://docs.microsoft.com/microsoftt
 It also shows Device permissions for the browser. Please refer [Device permissions for browser](https://docs.microsoft.com/microsoftteams/platform/concepts/device-capabilities/browser-device-permissions) for more information.
 
 ```
-The latest update on how device permissions are handled in the browser is currently available in public developer preview
-only.
-
-This update will be generally available (GA) by February 01, 2022
+Currently only capture image is supported in Teams Desktop
```
[the remark this answers is not in the document]
Comment (Prithvi-MSFT) [reply samples/tab-device-permissions/nodejs/README.md]: changed

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5314966459

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine.

**Review by Mohammed-MSFT — APPROVED** [review]
Changes Looks Fine Approving It.

**Pull request merged**